File: Java Problems/PWSkills Assignments/Java Operators/SumOfDigits.java
import java.util.Scanner;

public class SumOfDigits {
    public static void main(String[] args) {
        System.out.print("Enter a 3 digit number: ");
        Scanner sc = new Scanner(System.in);
        int number = sc.nextInt();
        int sum = (number%10);
        number /= 10;
        sum += number%10;
        number/=10;
        sum += number;
        System.out.print("Sum of the digits of this number is = " + sum);
    }
}

File: C++ Programs/Recursion/sumOfDigits.cpp
#include <iostream>
using namespace std;
int sum(int number)
{
    // Base Case
    if (number % 10 == number)
        return number;
    return number % 10 + sum(number / 10);
}
int main()
{
    int number;
    cin >> number;
    cout << sum(number) << endl;
    return 0;
}